<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyIPAddressEdit</class>
 <widget class="QWidget" name="MyIPAddressEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>5</y>
     <width>460</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="MyIpPartLineEdit" name="IPlineEdit1">
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="MyIpPartLineEdit" name="IPlineEdit2">
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="MyIpPartLineEdit" name="IPlineEdit3">
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="MyIpPartLineEdit" name="IPlineEdit4">
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>98</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyIpPartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>myipaddressedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
